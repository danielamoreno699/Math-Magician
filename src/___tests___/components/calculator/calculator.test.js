import { render } from '@testing-library/react';
import Calculator from '../../../components/calculator/calculator';

describe('Calculator', () => {
  test('renders correctly', () => {
    const { container } = render(<Calculator />);
    expect(container.querySelector('.calculator-text')).toBeTruthy();
  });
});
