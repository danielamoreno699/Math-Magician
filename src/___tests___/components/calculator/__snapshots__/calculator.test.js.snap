// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Calculator renders correctly 1`] = `
<div
  class="mt-5"
>
  <div
    class="container mt-5"
  >
    <div
      class="row d-flex justify-content-center"
    >
      <div
        class="col"
      >
        <h2
          class="calculator-text"
        >
          Let's do some math
        </h2>
      </div>
      <div
        class="col"
      >
        <div
          class="wrapper"
        >
          <div
            class="screen"
          >
            0
          </div>
          <div
            class="btn-box"
          >
            <button
              class="btn-number"
              type="button"
            >
              AC
            </button>
            <button
              class="btn-number"
              type="button"
            >
              +/-
            </button>
            <button
              class="btn-number"
              type="button"
            >
              %
            </button>
            <button
              class="btn-number"
              type="button"
            >
              รท
            </button>
            <button
              class="btn-number"
              type="button"
            >
              7
            </button>
            <button
              class="btn-number"
              type="button"
            >
              8
            </button>
            <button
              class="btn-number"
              type="button"
            >
              9
            </button>
            <button
              class="btn-number"
              type="button"
            >
              x
            </button>
            <button
              class="btn-number"
              type="button"
            >
              4
            </button>
            <button
              class="btn-number"
              type="button"
            >
              5
            </button>
            <button
              class="btn-number"
              type="button"
            >
              6
            </button>
            <button
              class="btn-number"
              type="button"
            >
              -
            </button>
            <button
              class="btn-number"
              type="button"
            >
              1
            </button>
            <button
              class="btn-number"
              type="button"
            >
              2
            </button>
            <button
              class="btn-number"
              type="button"
            >
              3
            </button>
            <button
              class="btn-number"
              type="button"
            >
              +
            </button>
            <button
              class="btn-number"
              type="button"
            >
              0
            </button>
            <button
              class="btn-number"
              type="button"
            >
              .
            </button>
            <button
              class="btn-number"
              type="button"
            >
              =
            </button>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
